src/schema/v2/homeView/getSectionsForUser.ts

```diff
@@ -1,83 +1,14 @@
 import { ResolverContext } from "types/graphql"
-import {
-  LatestActivity,
-  AuctionLotsForYou,
-  CuratorsPicksEmerging,
-  FeaturedFairs,
-  HeroUnits,
-  HomeViewSection,
-  LatestArticles,
-  MarketingCollections,
-  NewWorksForYou,
-  NewWorksFromGalleriesYouFollow,
-  RecentlyViewedArtworks,
-  RecommendedArtists,
-  ShowsForYou,
-  SimilarToRecentlyViewedArtworks,
-  TrendingArtists,
-  ViewingRooms,
-  LatestAuctionResults,
-  News,
-  Auctions,
-} from "./sections"
+import { HomeViewSection } from "./sections"
+import { getLegacyZoneSections } from "./zones/legacy"
 
 export async function getSectionsForUser(
   context: ResolverContext
 ): Promise<HomeViewSection[]> {
-  /*
-   * FAKE temporary placeholder logic for determining the sections that a user will see
-   */
-  const { meLoader } = context.authenticatedLoaders
+  const legacyZoneSections = await getLegacyZoneSections(context)
 
-  if (!meLoader) throw new Error("You must be signed in to see this content.")
-
-  const me = await meLoader()
-
-  let sections: HomeViewSection[] = []
-
-  if (me.type === "Admin") {
-    sections = [
-      LatestActivity,
-      LatestAuctionResults,
-      Auctions,
-      LatestArticles,
-      RecentlyViewedArtworks,
-      ShowsForYou,
-      MarketingCollections,
-      TrendingArtists,
-      FeaturedFairs,
-      CuratorsPicksEmerging,
-      SimilarToRecentlyViewedArtworks,
-      AuctionLotsForYou,
-      NewWorksForYou,
-      HeroUnits,
-      NewWorksFromGalleriesYouFollow,
-      RecommendedArtists,
-      ViewingRooms,
-      News,
-    ]
-  } else {
-    sections = [
-      News,
-      LatestActivity,
-      Auctions,
-      CuratorsPicksEmerging,
-      SimilarToRecentlyViewedArtworks,
-      ShowsForYou,
-      MarketingCollections,
-      FeaturedFairs,
-      NewWorksForYou,
-      HeroUnits,
-      AuctionLotsForYou,
-      LatestArticles,
-      RecommendedArtists,
-      TrendingArtists,
-      NewWorksFromGalleriesYouFollow,
-      RecentlyViewedArtworks,
-      LatestAuctionResults,
-      ViewingRooms,
-    ]
-  }
-
-  return sections
+  return [
+    ...legacyZoneSections,
+    // other zones’ sections TK
+  ]
```

Comment:
Trying to keep things simple for now. 

(This might be more apparent if you read this diff in **Split** mode, or view the [updated file](https://github.com/artsy/metaphysics/blob/721583d59ac4fca4957d0d1b950874a24a85ac4d/src/schema/v2/homeView/getSectionsForUser.ts).)